### ISSUE TITLE
nothing collected for netsh

### ISSUE THREAD

**Issue by lysShub** [lowleveldesign/wtrace#18]
```sh
C:\WINDOWS\system32>strace.exe -c -v  netsh interface ipv4 set address name="test"  static 169.254.215.40 255.255.0.0

wtrace v3.3.22257.11 - collects process or system traces
Copyright (C) 2022 Sebastian Solnica (lowleveldesign.org)
Visit https://wtrace.net to learn more

HANDLERS
  file, image, process, rpc, tcp, udp

Starting the tracing session (might take a moment). Press Ctrl + C to exit.
WTrace.ETW.Tracing Information: 0 : [etw] Starting main ETW session
WTrace.ETW.Tracing Information: 0 : [etw] Starting rundown session wtrace-rt_rundown
WTrace.Tracing Information: 0 : [filter] including process netsh (2672)
WTrace.Tracing Information: 0 : [filter] including process netsh (2672)
WTrace.ETW.Tracing Information: 0 : [etw] Rundown session finished

Process (2672) exited.
WTrace.ETW.Tracing Information: 0 : [etw] Main ETW session completed

--------------------------------
           Processes
--------------------------------
├─ netsh [2672]
C:\WINDOWS\system32>
```

**Comment by lowleveldesign** [comment]
I suspect that the reason is that there is not much going on after the main thread is created (all modules were already loaded etc.). Wtrace starts process as suspended and then starts the session. Therefore, it misses a set of initial events. I will try to fix that in the next release. Thanks for reporting.

**Comment by lysShub** [comment]
thx

**Comment by lowleveldesign** [comment]
@lysShub it was something different, but it should be fixed in version 3.4. Please give it a try. Thanks again for reporting.

**Comment by lysShub** [comment]
@lowleveldesign   
![image](https://github.com/lowleveldesign/wtrace/assets/55095917/47facbaa-9ca1-4f5b-9b8d-c650b845e06f)
GOOD JOD!
